2_zadacha.py

```diff
@@ -0,0 +1,21 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, ч
+# то Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+    # 60 -> 10  40  10
+    
+
+S = int (input ("vsego ="))
+a = int (S/6)
+b = int (S/6)
+c = int (S - (S/3))
```

Comment:
Функцию `int` использовать не обязательно. Существует прекрасный оператор целочисленного отделения.
```suggestion
a = S // 6
b = S // 6
c = S - S // 3
```